package/MDAnalysis/analysis/align.py

```diff
@@ -864,8 +873,6 @@ def __init__(self, mobile, reference=None, select='all', filename=None,
         # do this after setting the memory reader to have a reference to the
         # right reader.
         super(AverageStructure, self).__init__(mobile.trajectory, **kwargs)
-        if not self._verbose:
-            logging.disable(logging.WARN)
```

Comment:
Same question as above.

Reply:
same answer ;-) — yes, here it matters for as long as `__init__` is doing work.